src/EnergyPlus/HWBaseboardRadiator.cc

```diff
@@ -1592,20 +1588,19 @@ namespace HWBaseboardRadiator {
         int WaterInletNode;
         int WaterOutletNode;
         auto &Iter = state.dataHWBaseboardRad->Iter;
-        auto &MyEnvrnFlag2 = state.dataHWBaseboardRad->MyEnvrnFlag2;
         auto &HWBaseboard = state.dataHWBaseboardRad->HWBaseboard;
```

Comment:
auto const &HWBaseboard?

Reply:
Never reference the class, only the object. All of these are wrong since we shouldn't auto scalars and shouldn't reference them either.

Reply:
@rraustad, you mean "scalar" not "class".

Reply:
right, I missed that, MyEnvrnFlag2 is bool too